Runtime/Scripts/WebRTC.cs

```
@@ -230,6 +231,23 @@ public static class WebRTC
 
         public static void Initialize(EncoderType type = EncoderType.Hardware)
         {
+            if (Application.platform != RuntimePlatform.LinuxEditor &&
+                Application.platform != RuntimePlatform.LinuxPlayer)
+            {
+                if (SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLCore ||
+                    SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLES2 ||
+                    SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLES3)
+                {
+                    Debug.LogError($"Not Support OpenGL API on {Application.platform}");
+#if UNITY_EDITOR
+                    UnityEditor.EditorApplication.isPlaying = false;
+#else
+                    Application.Quit ();
```

It should be better to throw the exception to notice the error to developers.